tests/test_session.py

```diff
@@ -180,3 +184,143 @@ class TestSession(tb.QueryTestCase):
             )
         finally:
             await tx.rollback()
+
+    async def test_session_only_function_01(self):
```

Comment:
I don't think the positive tests are all that useful.  The session_only is applied to the whole query, so testing "various SELECT scenrarios" is redundant.

Reply:
Removed extra 2 tests.